lambda/test_events.py

```diff
@@ -1,13 +1,65 @@
-import os
+import json
 
-import marchon
+import requests_mock
 
+import marchon
 '''
     set these in environment
 
     GOOGLE_API_KEY
     SHEET_ID
     MAPBOX_ACCESS_TOKEN
 '''
+
+
 # print output instead of saving to S3
-marchon.events_lambda_handler(event=None, context=None, dry_run=True)
+def call_marchon_events_lambda_handler():
```

Comment:
why make a function that only calls a function? call the function directly instead